.github/workflows/build_deploy.yaml

```diff
@@ -0,0 +1,148 @@
+name: Continuous Integration and Deployment of B2AI Peds Protocol
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set Environment Variables
+      run: |
+        echo "PROTOCOL_COMBINED_SLUG=peds-protocol-combined" >> $GITHUB_ENV
```

Comment:
you can set env vars with an `env:` keyword: https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/use-variables